testing.md

```diff
@@ -7,22 +7,243 @@ recipes cover how to write automated tests and see their code
 coverage. We also give pointers to plug those in modern continuous
 integration services like Travis CI and Coveralls.
 
-We will use an established and well-designed regression testing
-framework called [Prove](https://github.com/fukamachi/prove). It is
-not the only possibility though,
-[FiveAM](http://quickdocs.org/fiveam/api) is a popular one (see
-[this blogpost](http://turtleware.eu/posts/Tutorial-Working-with-FiveAM.html) for an
-introduction) and [there are others](https://github.com/CodyReichert/awesome-cl#unit-testing) (and more again). We prefer
-`Prove` for its documentation and its **extensible reporters** (it has different
-report styles and we can extend them).
+[Prove](https://github.com/fukamachi/prove) used to be a widely liked testing framework but its repository was later archived. Its successor [Rove](https://github.com/fukamachi/rove) is not stable enough, so we will be using a mature testing framework called [FiveAM](https://github.com/lispci/fiveam). There are also some [other testing frameworks](https://github.com/CodyReichert/awesome-cl#unit-testing) to explore.
```

Comment:
You don't link to 5am docs! (they are not reachable from its Github)

Reply:
OH! My fault. It only has a API doc that I simply missed it. :-(

Reply:
Fixed.